runtime/config-deployer-service/ballerina/APIClient.bal

```diff
@@ -809,33 +821,53 @@ public class APIClient {
             operationPoliciesToUse = operation.operationPolicies;
         }
         if operationPoliciesToUse is APIOperationPolicies {
-            APKOperationPolicy[]? request = operationPoliciesToUse.request;
+            APKOperationPolicy[]? requestPolicies = operationPoliciesToUse.request;
+            APKOperationPolicy[]? responsePolicies = operationPoliciesToUse.response;
+            if requestPolicies is APKOperationPolicy[] && requestPolicies.length() > 0 {
```

Comment:
We can pull these declarations outside and combine the `if` conditions to reduce the nested levels